### ISSUE TITLE
Images in Documentation Not Showing Up

### ISSUE THREAD

**Issue by wbinventor** [mit-crpg/OpenMOC#286]
I've been updating some of the documentation on `openmoc.materialize` and realized that many of the images in the section on radial/angular discretization are not tracked by Git. There were some recent PRs by @geogunow and @samuelshaner to update this section - when you guys build the docs, do all of the images show up properly? Here is the list of warnings (really, errors) that I see when I run sphinx:

```bash
/home/wboyd/Documents/NSE-CRPG-Codes/OpenMOC/docs/source/usersguide/input.rst:402: WARNING: Duplicate explicit target name: "figa".
/home/wboyd/Documents/NSE-CRPG-Codes/OpenMOC/docs/source/usersguide/input.rst:402: WARNING: Duplicate explicit target name: "figb".
/home/wboyd/Documents/NSE-CRPG-Codes/OpenMOC/docs/source/usersguide/input.rst:428: WARNING: Duplicate explicit target name: "figa".
/home/wboyd/Documents/NSE-CRPG-Codes/OpenMOC/docs/source/usersguide/input.rst:428: WARNING: Duplicate explicit target name: "figb".
/home/wboyd/Documents/NSE-CRPG-Codes/OpenMOC/docs/source/usersguide/input.rst:457: WARNING: Duplicate explicit target name: "figa".
/home/wboyd/Documents/NSE-CRPG-Codes/OpenMOC/docs/source/usersguide/input.rst:457: WARNING: Duplicate explicit target name: "figb".
/home/wboyd/Documents/NSE-CRPG-Codes/OpenMOC/docs/source/usersguide/input.rst:632: WARNING: Duplicate explicit target name: "figa".
/home/wboyd/Documents/NSE-CRPG-Codes/OpenMOC/docs/source/usersguide/input.rst:632: WARNING: Duplicate explicit target name: "figb".
/home/wboyd/Documents/NSE-CRPG-Codes/OpenMOC/docs/source/usersguide/input.rst:670: WARNING: Duplicate explicit target name: "figa".
/home/wboyd/Documents/NSE-CRPG-Codes/OpenMOC/docs/source/usersguide/input.rst:670: WARNING: Duplicate explicit target name: "figb".
/home/wboyd/Documents/NSE-CRPG-Codes/OpenMOC/docs/source/usersguide/input.rst:None: WARNING: image file not readable: usersguide/../../img/annular_pin_materials.png
/home/wboyd/Documents/NSE-CRPG-Codes/OpenMOC/docs/source/usersguide/input.rst:None: WARNING: image file not readable: usersguide/../../img/annular_pin_inner_rings.png
/home/wboyd/Documents/NSE-CRPG-Codes/OpenMOC/docs/source/usersguide/input.rst:None: WARNING: image file not readable: usersguide/../../img/annular_pin_fuel_rings.png
/home/wboyd/Documents/NSE-CRPG-Codes/OpenMOC/docs/source/usersguide/input.rst:None: WARNING: image file not readable: usersguide/../../img/annular_pin_outer_rings.png
```

**Comment by wbinventor** [comment]
Btw, I think the first several set of warnings must be related to those recent PRs - perhaps one of you copy-pasted a figure as a template for several new figures without updating the name of the figure link.

**Comment by samuelshaner** [comment]
This is probably due to my PR with updating the ringify to use equal radius rings for moderator rings. I'll look into this now.

**Comment by wbinventor** [comment]
Closed with #278.
